source/Services/atlassian-infrastructure.html.md.erb

```diff
@@ -0,0 +1,133 @@
+
+There are total 9 VMs in the production environment, 3 VMs for Jira, 2 VMs for Confluence, 1 VM for Crowd and 3 VMs for GlusterFS.
+
+All resources are stored in either [atlassian-prod-rg](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/79898897-729c-41a0-a5ca-53c764839d95/resourceGroups/atlassian-prod-rg/overview) or [atlassian-nonprod-rg](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/b7d2bd5f-b744-4acc-9c73-e068cec2e8d8/resourceGroups/atlassian-nonprod-rg/overview)
+
+```bash
+Prod
+
+#### Jira VMs
+* atlassian-prod-jira-01
+* atlassian-prod-jira-02
+* atlassian-prod-jira-03
+
+#### Confluence VMs
+* atlassian-prod-confluence-02
+* atlassian-prod-confluence-04
+
+#### Crowd VM - this is used to handle the authentication.
+* atlassian-prod-crowd-01
+
+#### GlusterFS VMs - GlustFS is used to store file attachments.
+* atlassian-prod-gluster-01
+* atlassian-prod-gluster-02
+* atlassian-prod-gluster-03
+
+#### Postgres Flexible Server
+atlassian-prod-flex-server
+
+staging / non prod VM's following the same naming convention but with "nonprod" instead of "prod"
+```
+
+Please note that there are other important resources like Application Gateways, Load Balancer, NSGs, VNETs, SendGrid etc. which are not mentioned here.
+
+## PlatOps responsibility
+
+The internal tooling team have access to the servers and databases in both production and staging, as well as in-depth knowledge of how the Atlassian apps work. This means that self-service is normally possible.
+
+What cant the internal team do? 
+
+The team does not have access to Azure, or the automation scripts. This means that PlatOps will need to provide assistance in these spaces. This includes any network related issues or features.
+
+## Accessing the VMs
+1. Connect to the F5 VPN - available at https://portal.platform.hmcts.net/
+
+2. Download the private key "test-private-key" from the key vault [atlassian-nonprod-kv](https://portal.azure.com/#@HMCTS.NET/asset/Microsoft_Azure_KeyVault/Secret/https://atlasssian-prod-kv.vault.azure.net/secrets/test-private-key), and save it to a file.  
+
+3. Set the correct permissions on the file and 
+```bash
+run chmod 600 <privatekeyfilename>
+```
+
+4. SSH to the desired VM.
+```bach
+ssh -i <privatekeyfilename> atlassian-admin@<VM-IP>
+```
+
+###NOTE: as the VM's are restores of a previous production environment, the local hostname will be different from the Azure display name. Verify you are on the right host by its local IP address!
+
+For example Jira node 1 in staging and production will both the same hostname as "PRDATL01AJRA01.cp.cjs.hmcts.net"
+
+## Auto SSL renewal process
+
+The environments have been built to use Lets Encrypt SSL certificates via the [acmebot](https://acmedtssdsprod.azurewebsites.net/dashboard)
+
+This means that the certificates renew  regularly (every 3 months) so automated has been created to automatically renew these certs within the environment.
+
```

Comment:
Minor grammar gripe, it's not quite reading right. Should be `automation` instead of `automated`